src/components/Routes/routesUrl.ts

```diff
@@ -0,0 +1,5 @@
+export const homeUrl = '/';
+export const eventsUrl = '/events';
+export const agenciesUrl = '/agencies';
+export const astronautsUrl = '/astronauts';
+export const aboutUrl = '/about';
```

Comment:
1. Please use `enum` here
2. Rename the file to `routesPath`

Reply:
1. done
2. done